pkgs/servers/http/openresty/default.nix

```diff
@@ -1,25 +1,32 @@
-{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, libxml2, libxslt, gd, geoip
-, perl }:
+{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, postgresql, libxml2, libxslt,
+gd, geoip, perl }:
 
 assert stdenv.isLinux;
```

Comment:
This line can be removed. `platforms` is enough.

Reply:
Thanks for review. Removed.